learn/cookbooks/docker.md

```diff
@@ -0,0 +1,123 @@
+# Using Meilisearch with Docker
+
+Docker is a tool that packages applications into containers portable across a wide range of environments. When using Docker for development, we recommend following [the official instructions to install Docker Desktop](https://docs.docker.com/get-docker/).
+
+In this guide you will learn how to use Docker to download and run Meilisearch, configure your instance, and manage your Meilisearch data.
+
+## Download Meilisearch with Docker
+
+Docker containers are distributed in images. To use Meilisearch, use the `docker pull` command to download a Meilisearch image:
+
+```sh
+docker pull getmeili/meilisearch:v0.28
```

Comment:
If this is merged after v0.29, we need to update it. Or we could go with `getmeili/meilisearch:v0.X.X` so we don't have to update it every time

Reply:
I would say `vX.Y.Z` is more common, and your guide will be already ready for v1 😇

Reply:
I was thinking, we could consider automating the version update in documentation for every release as we already do on the integration team side
An automated PR will be created to update all the `v0.28` into `v0.29` for example

@dichotommy come to me if you want to introduce this automation, I think we could create something quite easily with a GitHub action
It would be the best solution: nothing to maintain on your side, and the users can copy/paste

Reply:
That's an interesting idea. I had planned to look into string interpolation for this (e.g. we have a variable in config.js and write this sample as `getmeili/meilisearch:${version}`). 

Do you think the automated PR route could cause problems when we start giving access to different documentation versions to users?

Reply:
When you will version the docs, we will need to adapt the automation, definitely. Since I cannot how you will handle it (with git branch? With different repos?) I cannot guess.
In we don't know how to adapt it, we will remove it, this is the best way to "adapt" haha